bc_obps/reporting/schema/report_operation.py

```diff
@@ -29,8 +32,23 @@ class Meta:
             'bc_obps_regulated_operation_id',
             'activities',
             'regulated_products',
-            'operation_representative_name',
         ]
+        orm_mode = True
+
+
+class ReportOperationRepresentativeSchema(ModelSchema):
+    class Meta:
+        model = ReportOperationRepresentative
+        fields = ["id", "representative_name", "selected_for_report"]
+
+
+class ReportOperationSchemaOut(Schema):
```

Comment:
Instead of creating a whole extra `ReportOperationSchemaOut` class, we could extend the existing `ReportOperationOut` and add the `report_operation_representatives: List[...]` field on it